crates/containerd-shim-wasm/src/sandbox/shim/local.rs

```diff
@@ -34,6 +37,51 @@ use crate::sys::metrics::get_metrics;
 #[cfg(test)]
 mod tests;
 
+#[derive(Message, Clone, PartialEq)]
+struct Options {
+    #[prost(string)]
+    type_url: String,
+    #[prost(string)]
+    config_path: String,
+    #[prost(string)]
+    config_body: String,
+}
+
+#[derive(Deserialize, Clone, PartialEq)]
+struct Config {
+    #[serde(alias = "SystemdCgroup")]
+    systemd_cgroup: bool,
+}
+
+impl Default for Config {
+    fn default() -> Self {
+        Self {
+            systemd_cgroup: true,
```

Comment:
IIUC what runc is doing, I think this should default to `false`.
Also, I think defaulting this to false would fix a few of the CI failures.

Reply:
If you set it to false, most k8s clusters will have metrics bugs unles explicitly specify `SystemdCgroup = true` in containerd/config.toml.
Is this OK?

> Also, I think defaulting this to false would fix a few of the CI failures.

I think the kind problem will be solved.
But when using systemd cgroups, kind pod doesn't seem to work.
As you can see below, kind supports systemd cgroups. It's strange that it doesn't work.
https://github.com/containerd/runwasi/pull/356#issuecomment-1766693160

Reply:
> Is this OK?

IIUC, that would be the same behaviour as runc, so it´s fine with me.

Reply:
Also, I think it's better to err on the side of not using systemd.
If we wrongly DON'T use systemd, the metrincs don't work.
If we wrongly DO use systemd, the container doesn't even start.

Reply:
I don't understand it well enough.
Does this mean that if containerd has `SystemdCgroup = false` and runwasi uses systemd, the container won't start?

Reply:
Exactly